src/App.tsx

```diff
@@ -43,6 +46,20 @@ function getWsUri(id: string) {
   );
 }
 
+function useKeyboardCtrlIntercept(key: string, reaction: (evt: KeyboardEvent) => unknown) {
+  useEffect(() => {
+    const wrappedReaction: typeof reaction = (evt) => {
+      if (!(evt.ctrlKey && evt.key.toLowerCase() === key.toLowerCase())) return;
```

Comment:
The Ctrl+S keyboard shortcut doesn't seem to quite work on macOS. I would expect it to map to Cmd+S, but it doesn't work.

Also, you don't need to negate the `if` condition and return, probably clearer to just write a direct conditional.

Reply:
There seems to also be KeyboardEvent.metaKey, I'll just or that with ctrlKey. That will lead to windows + S also working, but I guess that wont hurt.